lib/api.js

```diff
@@ -391,9 +401,35 @@ function _addNewLedgerAgent(actor, ledgerNodeId, options, callback) {
         callback(err);
       });
     },
-    getLedgerNode: ['checkPermission', (results, callback) =>
-      brLedgerNode.get(actor, ledgerNodeId, options, callback)
-    ],
+    // validate all the specified plugins
+    plugins: callback => {
+      let pluginName;
+      try {
+        for(pluginName of ledgerAgent.plugins) {
+          const p = brLedgerNode.use(pluginName);
+          if(p.type !== 'ledgerAgentPlugin') {
+            throw new Error('The plugin `type` must be `ledgerAgentPlugin`.');
+          }
+          if(!p.api.serviceType) {
+            throw new Error('`serviceType` must be defined.');
+          }
+          if(!p.api.router) {
+            throw new Error('`router` must be defined.');
+          }
+          if(!p.api.router.stack.some(s => s.route.path === '/')) {
+            throw new Error('A root route `/` must be defined.');
+          }
+        }
+      } catch(e) {
+        return callback(new BedrockError(
+          'Invalid ledger agent plugin.', 'InvalidAccessError',
```

Comment:
`InvalidAccessError` doesn't seem right here ... should be a `TypeError` or `SyntaxError`, IMO.

Reply:
changed to SyntaxError.